server/pulp/server/webservices/urls.py

```diff
@@ -23,11 +27,12 @@
                                                            ConsumerGroupBindingsView,
                                                            ConsumerGroupContentActionView,
                                                            ConsumerGroupResourceView,
+                                                           ConsumerGroupSearchView,
                                                            ConsumerGroupUnassociateActionView,
                                                            ConsumerGroupView)
```

Comment:
I know this PR isn't the place to change this, but I'm not a fan of this style of importing. I wish we had just done

```
from pulp.server.webservices.views import consumers
```